clients/csharp-dotnet-core/voice-assistant-test/tool/MainService.cs

```diff
@@ -206,80 +204,174 @@ private static async Task<bool> ProcessTestFiles(AppSettings appSettings)
                 // Keep track of high-level (summary) results for all dialogs in a single input test file. This list will be serialized to JSON as part of the overall single test report.
                 List<DialogReport> dialogReports = new List<DialogReport>();
 
-                foreach (Dialog dialog in fileContents)
+                bool sendFirst = true;

```

Comment:
sendFirst is not needed. Just hard code your true/false values when you call the function

Reply:
Done